packages/composites/vesper/schemas/env.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/example-composite-adapter.json",
+  "title": "@chainlink/example-composite-adapter env var schema",
+  "required": ["RPC_URL"],
+  "type": "object",
+  "properties": {
+    "RPC_URL": {
+      "type": "string",
+      "format": "uri"
+    },
+    "OPTION": {
```

Comment:
What is this one?

Reply:
I'm out of the habit of updating this file, so it's just the auto-generated content. Fixing now